reference/content/whats-new.md

```diff
@@ -202,7 +202,7 @@ Key new features of the 3.8 Java driver release:
 ### Transactions
 
 The Java driver now provides support for executing CRUD operations within a transaction (requires MongoDB 4.0).  See the
-[Transactions and MongoDB Drivers](https://docs.mongodb.com/master/core/transactions/#transactions-and-mongodb-drivers) section
+[Transactions and MongoDB Drivers](https://www.mongodb.com/docs/upcoming/core/transactions/#transactions-and-mongodb-drivers) section
```

Comment:
issue: I wonder if `manual` rather than `upcoming` would be more appropriate (https://www.mongodb.com/docs/manual/core/transactions/#transactions-and-mongodb-drivers)

Reply:
Never mind, I guess upcoming is aliased to 5.3, rapid to 5.2, no-version to 5.0, etc.